### PR TITLE
feat(pii): Add custom placeholder field

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Description:** 

* Add a text field for replace to take a custom placeholder text other than [Filter]. The resulting JSON should look like this: {"type": "creditcard", "redaction": {"method": "replace", "text": "[Filter]"}}

This is the second step of this task: https://app.asana.com/0/1158284503473033/1169336924425598

**Preview:**

image

Next Steps:

- Make event ID input a button
- Replace Modals with the GlobalModal component

### CODE REVIEW

**Review by untitaker — COMMENTED** [review]
It looks like this to me (with a large viewport):

Screenshot 2020-06-16 at 13 54 02

Can we add spacing after every second field here?

File: src/sentry/static/sentry/app/views/settings/components/dataScrubbing/rulesList.tsx [hunk 1]
```diff
@@ -17,22 +17,42 @@ type Props = {
...
+  descriptionDetails.push(`[${typeLabel}]`);
+
+  if (rule.type === RuleType.PATTERN) {
+    descriptionDetails[descriptionDetails.length - 1] =
+      descriptionDetails.length === 3
+        ? `${t('using')} [${rule.pattern}]`
+        : `[${rule.pattern}]`;
+  }
+
+  return `${descriptionDetails.join(' ')} ${t('from')} [${source}]`;
```
Comment (untitaker) [inline src/sentry/static/sentry/app/views/settings/components/dataScrubbing/rulesList.tsx:42]: If I have a regex replace rule, I get:

```Replace with [b] using [a] from [**]```

where I would've expected:

```Replace [a] with [b] from [**]```

**Review by untitaker — APPROVED** [review]
works

**Comment by priscilawebdev** [comment]
As we talked  I will group the fields in a follow-up PR 😉

**Pull request merged**